libs/server-sdk/src/data_store/data_store_updater.hpp

```diff
@@ -0,0 +1,93 @@
+#pragma once
+
+#include "../data_source/data_source_update_sink.hpp"
+#include "data_store.hpp"
+#include "dependency_tracker.hpp"
+
+#include <launchdarkly/server_side/change_notifier.hpp>
+
+#include <boost/signals2/signal.hpp>
+
+#include <memory>
+
+namespace launchdarkly::server_side::data_store {
+
+class DataStoreUpdater
```

Comment:
I wonder why my local linter doesn't complain about this. Time to default all the things!

Reply:
So, the thing that was here, where the linter was just exploding, is gone now. Which is maybe why I got other new lints.